contracts/zero-ex/test/features/native_orders_feature_test.ts

```diff
@@ -82,6 +96,19 @@ blockchainTests.resets('NativeOrdersFeature', env => {
             env.txDefaults,
             artifacts,
         );
+        // contract wallet for signer delegation
+        contractWallet = await TestSignerRegistryWithContractWalletContract.deployFrom0xArtifactAsync(
+            artifacts.TestSignerRegistryWithContractWallet,
+            env.provider,
+            env.txDefaults,
```

Comment:
Instead of using `env.txDefaults.from!` everywhere, you can use another unlocked address, i.e. on line 63:
```
[owner, maker, taker, notMaker, notTaker, contractWalletSigner] = await env.getAccountAddressesAsync();
```

Then deploy `contractWallet` from `contractWalletSigner` like this:
```suggestion
            { ...env.txDefaults, from: contractWalletSigner },
```